src/StatusQ/Components/StatusChatListItem.qml

```diff
@@ -0,0 +1,195 @@
+        }
+
+        Component {
+            id: statusLetterIdenticonCmp
+            StatusLetterIdenticon {
+                height: 24
+                width: 24
+                name: statusChatListItem.name
+                letterSize: 15
+            }
+        }
+
+        Component {
+            id: statusRoundedImageCmp
+            Item {
+                height: 24
+                width: 24
+                StatusRoundedImage {
+                    id: statusRoundedImage
+                    width: parent.width
+                    height: parent.height
+                    image.source: statusChatListItem.image.source
+                }
+
+                Loader {
+                    sourceComponent: {
+                        switch (statusRoundedImage.image.status) {
+                            case Image.Loading:
+                              return statusLoadingIndicator
+                              break;
+                            case Image.Error:
+                              return statusLetterIdenticonCmp
+                              break;
+                        }
+                    }
+                }
+
+                Component {
+                    id: statusLoadingIndicator
+                    StatusLoadingIndicator {
+                        color: Theme.palette.directColor6
+                    }
+                }
+            }
+        }
+
+        StatusIcon {
+            id: statusIcon
+            anchors.left: identicon.right
+            anchors.leftMargin: 8
+            anchors.verticalCenter: parent.verticalCenter
+
+            width: 14
+            visible: statusChatListItem.type !== StatusChatListItem.Type.OneToOneChat
+            opacity: {
+                if (statusChatListItem.muted && !sensor.containsMouse) {
+                    return 0.4
+                }
+                return statusChatListItem.hasMention || 
+                  statusChatListItem.hasUnreadMessages || 
+                  statusChatListItem.selected ||
+                  statusBadge.visible ||
+                  sensor.containsMouse ? 1.0 : 0.7
+            }
+
+            icon: {
+                switch (statusChatListItem.type) {
+                    case StatusChatListItem.Type.PublicCat:
+                        return Theme.palette.name == "light" ? "public-chat" : "public-chat-white"
+                        break;
+                    case StatusChatListItem.Type.GroupChat:
+                        return Theme.palette.name == "light" ? "group" : "group-white"
+                        break;
+                    case StatusChatListItem.Type.CommunityChat:
+                        return Theme.palette.name == "light" ? "channel" : "channel-white"
+                        break;
+                    default:
+                        return Theme.palette.name == "light" ? "public-chat" : "public-chat-white"
+                }
+            }
```

Comment:
This is rather unfortunate, however I couldn't find a better way to make this work. 

Here's the issue:

When using `StatusIcon` with a `color` with use a `ColorOverlay` to render the supplied `color` over the actual SVG (which comes with its own color [black, most of the time]).

This is problematic when supplying colors with alpha channel, such as the designs foresee here. When the color used for the `ColorOverlay` has transparency, the underlying SVG color with shine through, resulting in undesired colors. 

I tried making the `Image` in `StatusIcon` invisible using `opacity: 0` in case a `color` is supplied (so that only the `ColorOverlay` is rendered) but that makes the overlay invisible as well.

So the solution now is to use in deed 100% black and white icons and apply the `opacity` to `StatusIcon` and disabling the `ColorOverlay` when no `color` is supplied (which is the case here).